core/templates/pages/about-page/charts/bar-chart.component.css

```
@@ -44,93 +53,113 @@
+  background-color: var(--color);
+  border-radius: 2px;
+  display: flex;
+  height: 24px;
+  justify-content: flex-end;
+  min-height: 24px;
+  padding: 0;
+  width: calc(100% * var(--size, 1));
+}
+
+.oppia-bar-chart .oppia-bar-chart-table .data {
+  font-family: 'Roboto', Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)